Sources/MapboxCoreNavigation/Router.swift

```diff
@@ -167,6 +167,15 @@ public protocol Router: CLLocationManagerDelegate {
     func updateRoute(with indexedRouteResponse: IndexedRouteResponse,
                      routeOptions: RouteOptions?,
                      completion: ((Bool) -> Void)?)
+
+    /**
+     Asynchronously removes currently active route.
+
+     This method is called automatically by `NavigationViewController` at the end of the navigation.
+     If you use `Router` without `NavigationViewController`, you are supposed to call this method when you are done
+     with `Router`.
+     */
+    func removeRoute(completion: (() -> Void)?)
```

Comment:
Should the completion handler accept a Boolean-typed success argument, similar to `updateRoute(with:options:completion:)`? It may not be necessary now, but it would be more difficult to introduce later.

Reply:
I agree, thanks.

Reply:
Although it shouldn't fail I would say.

Reply:
Done